clients/egui/src/account/mod.rs

```diff
@@ -487,6 +487,67 @@ impl AccountScreen {
         }
     }
 
+    pub fn reload_markdown_tabs(&self, ctx: &egui::Context) {
```

Comment:
there's duplicate code between `reload_markdown_tabs` and `refresh_tree_and_workspace`, not sure if i should merge the two functions and add conditionals to avoid running unnecessary code (like refreshing the tree, when i only need to reload the tabs) or if i can tolerate the duplication

Reply:
Am I understanding correctly that you reload the tabs if the toolbar is toggled on or off?

If that's the case it feels like a very heavy weight way to get the toolbar to disappear. 

Ideally you would reach into each toolbar and toggle it's visibility and request a new frame, instead of re-reading all the files. 

let me know if I'm reading the code wrong

Reply:
correct, the tab reloads after the toolbar vis change. I modified it so that visibility changes from within without reloading.